inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSource.java

```diff
@@ -58,6 +58,9 @@ public class MongoDBSource extends StreamSource {
     @ApiModelProperty("MongoDB collection name")
     private String collection;
 
+    @ApiModelProperty("Debezium snapshot mode")
```

Comment:
comment suggest like this “Debezium snapshot mode. Available options are initial and never, default is initial” in MongoDBSourceRequest.java

Reply:
Done.